recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
     - geos
     - glib
     - ghostscript
+    - libgdal-jp2openjpeg
```

Comment:
`libgdal-jp2openjpeg` should be added as a `run` dependency.

Reply:
Oh yes! Good thing we had a test to catch this!